com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs

```diff
@@ -204,7 +204,8 @@ internal static void SetStereoViewAndProjectionMatrices(CommandBuffer cmd, Matri
             }
             else
             {
-                cmd.SetRenderTarget(destination, colorLoadAction, colorStoreAction, depthLoadAction, depthStoreAction);
+                ScriptableRenderer.SetRenderTarget(cmd, destination, colorLoadAction, colorStoreAction, ClearFlag.None, Color.black);
+                //cmd.SetRenderTarget(destination, colorLoadAction, colorStoreAction, depthLoadAction, depthStoreAction);
```

Comment:
Should this still be here?

Reply:
converted this to Draft PR as it needs more work.